modelon/impact/client/entities/project.py

```diff
@@ -350,6 +419,19 @@ def get_contents(self) -> List[ProjectContent]:
             for content in self._project_definition.content
         ]
 
+    def get_content_by_name(self, name: str) -> Optional[ProjectContent]:
```

Comment:
We could probably also add a `get_modelica_library_by_name` to be more specific:
```
def get_modelica_library_by_name(self, name: str):
  return self.get_content_by_name(name, ContentType.MODELICA)
```